mlir-tensorrt/executor/lib/Conversion/MemRefToExecutor.cpp

```diff
@@ -548,6 +549,21 @@ void executor::populateMemRefToExecutorPatterns(
 }
 
 namespace {
+
+class RemoveNoOpClonePattern : public OpRewritePattern<bufferization::CloneOp> {
+public:
```

Comment:
This is still illegal, we can't keep this here.

Reply:
Agreed. Please let me know how this can be better implemented.

Reply:
@christopherbate Please re-review as I have addressed your comment.